### DISCUSSION TITLE
Issue with plugin-client-redirects

### DISCUSSION THREAD

**Discussion by djipco** [facebook/docusaurus#6154] [Q&A] [answered]
I have installed **plugin-client-redirects** and set up a super simple redirection but it does not kick in. Is there anything special to do besides installing the plugin and adding the configuration ?

Here is my plugin config in `docusaurus.config.js`: 

```js
[
  "@docusaurus/plugin-client-redirects",
  {
    redirects: [
      {
        from: ["/test"],
        to: "/api",
      },
    ],
  },
],
```

**Comment by Josh-Cena** [comment, marked as answer]
Hi, did you make a production build? The plugin doesn't work in local development.

**Comment by djipco** [comment]
Oh, it doesn't? I did not know. Let me try a build.

**Comment by djipco** [comment]
Yeah... it works fine. My bad. When I went back to the documentation, I realized the info is right there at the top. Maybe it should be in a WARNING box instead of a NOTE box. Or maybe I should just improve my reading skills. Sorry for the trouble.

**Reply by Josh-Cena** [reply]
Yeah, it should probably be more visible. I'm planning to normalize the admonitions in our documentation soon
